packages/rest/src/writer.ts

```diff
@@ -50,8 +50,42 @@ export function writeResultToResponse(
         // TODO(ritch) remove this, should be configurable
         // See https://github.com/loopbackio/loopback-next/issues/436
         response.setHeader('Content-Type', 'application/json');
+        // let customResult = result;
```

Comment:
why is this commented ? can you check please ?

Reply:
This is a mistake. I'd fix it.